furusiya/model/maps/area_map.py

```diff
@@ -34,7 +34,7 @@ def is_walkable(self, x, y):
                         ]) == 0)
 
     def is_visible_tile(self, x, y):
-        return self.is_walkable(x, y) and not self.tiles[x][y].block_sight
+        return self.is_on_map(x, y) and not self.tiles[x][y].block_sight
```

Comment:
I don't agree with this. It means everything is visible unless it explicitly blocks sight. Things that are not walkable should not be visible. Please revert it, and set monsters to `block_sight = False` instead.

Reply:
Let me just post a line of code:
https://github.com/nightblade9/furusiya/blob/342ec2e51eedbd70fcd58672b04d1ea6729b39da/furusiya/model/maps/area_map.py#L71
A tile blocks sight in two cases:
- It is explicitly made to do so.
- It isn't walkable and hasn't been explicitly made not sight-blocking.

So by default, non-walkable tiles block sight *unless* explicitly allowed. Sounds better?

Reply:
Yes, sorry. This is fine.